azerothcore/azerothcore-wotlk
# fix(DB/SAI): Move Floon,Raliq the Drunk and Sal'salabim to SAI.

Coosh'coosh - Part of Quest Crackin' Some Skulls(10009) is fixed here https://github.com/azerothcore/azerothcore-wotlk/pull/15504
## Changes Proposed:
- Move the 3 npcs to SAI  

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4969
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15024
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13898
- Closes https://github.com/chromiecraft/chromiecraft/issues/5186


## SOURCE:

https://github.com/TrinityCore/TrinityCore/pull/20630/files
Co-authored-by: aletson <aletson@users.noreply.github.com>
## Tests Performed:

- tested

## How to Test the Changes:

evade move = Leaving combat and speaking with the npc again, which causes them to move their home pos, until a server restart. Check if creature is hostile up until he returns to their home pos, only then can you talk with it to start again.
(Using .gm on let's you speak with them even if they are hostile, don't use it to speak with them, while returning to home pos)

Sal'salabim
.go c id 18584
Check for gossip, should be nothing
.q a 10004
Check for gossip, should be "Altruis..." -> Click it, check if he talks when fight starts
Check if he does magnetic pull on 8-40y
Confirm that he CAN'T evade move
Check if he gives credit and resets at 20% health

Raliq the Drunk
.go c id 18585
Check for gossip, should be none
.q a 10009
Check for gossip, should be "I have been sent by..." -> Click it, check if he talks when fight starts
Check if he does uppercut
Confirm that he CAN'T evade move

Floon
.go c id 18588
Check for gossip, should be none
.q a 10009
Check for gossip , should be "I have been sent by..." -> "He wants his "golds" -> Click it,, check if he talks when fight starts
Confirm he casts frostbolt, frostnova and Silence
Confirm that he CAN'T evade move

## TODO

https://user-images.githubusercontent.com/46330494/226390411-7e016cfd-ae39-4325-805e-a8c2f1a9838f.mp4



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
